### PR TITLE
Updating READMEs

### PR DESCRIPTION
**Pull request by autodidaddict**
Signed-off-by: Kevin Hoffman <autodidaddict@users.noreply.github.com>

### CODE REVIEW

File: nats/README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-subscribes to the topic "example.actor" and the topic "example.task" in the queue group "work_queue".
-- `URI` nats connection uri. If not specified, the default is '0.0.0.0:4222'
-- `CLIENT_JWT` jwt auth token. For jwt authentication, both CLIENT_JWT and CLIENT_SEED must be provided.
-- `CLIENT_SEED` private seed for jwt authentication.
-
-For examples of invoking the messaging api, see
-- The example actor [nats-messaging](https://github.com/wasmCloud/examples/tree/main/actor/nats-messaging)
+| Property | Description |
+| :--- | :--- | 
+| `SUBSCRIPTION` | A comma-separated list of subscription topics. If a subscription is a queue subscription, follow the subscription with "\|" and the queue group name. For example, the setting `SUBSCRIPTION=example.actor,example.task\|work_queue` subscribes to the topic `example.actor` and the topic `example.task` in the queue group `work_queue`. |
```
Comment (stevelr) [inline nats/README.md:9]: in my markdown preview, the \ escape before the bar in double quotes works correctly , so just the bar shows up in preview, but the \ after 'example.task' is visible

Comment (stevelr) [reply nats/README.md:9]: I like the table format!

File: httpclient/README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
-# HTTP client capability provider
+# HTTP Client Capability Provider
 
-This capability provider implements the "wasmcloud:httpclient" capability
-contract using the rust 'reqwest' library
+This capability provider implements the `wasmcloud:httpclient` capability contract using the Rust [reqwest](https://docs.rs/reqwest) library.
```
Comment (stevelr) [inline httpclient/README.md:3]: This capability provider implements the `wasmcloud:httpclient` capability contract, and enables an actor to make outgoing http(s) requests. It is implemented in Rust using the [reqwest](https://docs.rs/reqwest) library.

File: httpserver-rs/README.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
-# wasmcloud HTTP Server Provider
+# wasmCloud HTTP Server Provider
 
-This library is a _native capability provider_ for the `wasmcloud:httpserver` 
-capability. Only actors signed with this capability 
-privilege will be allowed to use it. The actor that implements this contract 
-must implement the [HandleRequest](https://docs.rs/wasmcloud-interface-httpserver/0.1.5/wasmcloud_interface_httpserver/trait.HttpServer.html#) operation
+This executable is an implementation of the `wasmcloud:httpserver` capability. Only actors signed with this capability contract ID will be allowed to use it.
```
Comment (stevelr) [inline httpserver-rs/README.md:3]: This capability provider implements the `wasmcloud:httpserver` capability contract, and enables an actor to accept incoming http(s) requests. It is implemented in Rust with the [warp](https://docs.rs/warp/) web server framework and the fast and scalable [hyper](https://docs.rs/hyper/) http implementation.

File: README.md [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
-  - bulit in rust with Redis
+## Built-in Capability Providers
+The following capability providers are included automatically in every host runtime:
 
+| Provider | Contract | Description |
+| :--- | :--- | :--- |
+| **N/A** | [`wasmcloud:builtin:numbergen`](https://github.com/wasmCloud/interfaces/tree/main/numbergen) | Number generator, including random numbers and GUID strings |
+| **N/A** | [`wasmcloud:builtin:logging`](https://github.com/wasmCloud/interfaces/tree/main/logging) | Basic level-categorized text logging capability |
+
+While neither of these providers requires a _link definition_, to use either of them your actors _must_ be signed with their contract IDs.
```
Comment (stevelr) [inline README.md:27]: ## Additional Examples

Additional capability provider examples and sample code can be found in the [wasmCloud examples](https://github.com/wasmCloud/examples/tree/main/provider) repository.

File: nats/README.md (hunk 1 above)
Comment (autodidaddict) [reply nats/README.md:9]: Odd, in my markdown preview, the `\` does _not_ show up.

Comment (autodidaddict) [reply nats/README.md:9]: This is how github renders it:
image

**Review by stevelr — APPROVED** [review]

**Pull request merged**